container-chains/runtime-templates/frontier/src/migrations.rs

```diff
@@ -84,15 +132,18 @@ where
         let migrate_xcm_executor_utils_v4 =
             pallet_xcm_executor_utils::migrations::MigrateToV1::<Runtime>(Default::default());
         let migrate_pallet_xcm_v4 = MigrateToLatestXcmVersion::<Runtime>(Default::default());
+        let migrate_precompile_xcm_code =
+            MigratePrecompileXcmDummyCode::<Runtime>(Default::default());
         vec![
             // Applied in runtime 400
             // Box::new(migrate_precompiles),
-            Box::new(migrate_polkadot_xcm_v1),
+/*             Box::new(migrate_polkadot_xcm_v1),
             Box::new(migrate_xcmp_queue_v2),
             Box::new(migrate_xcmp_queue_v3),
-            Box::new(migrate_xcmp_queue_v4),
+            Box::new(migrate_xcmp_queue_v4), */
```

Comment:
Should we keep these migrations? I think the following ones were already applied in runtime 600:

- migrate_polkadot_xcm_v1
- migrate_xcmp_queue_v2
- migrate_xcmp_queue_v3
- migrate_xcmp_queue_v4

(correct me if I'm wrong)

Reply:
yes we can remove them